src/errors.jl

```diff
@@ -3,7 +3,7 @@ struct DistributionNotImplemented <: Exception
     gen_firstarg::Type
     func::Function
     modeltype::Type
-    dep_argtypes::NamedTuple
+    dep_argtypes::Union{AbstractVector, Core.SimpleVector}
```

Comment:
what is `Core.SimpleVector`?

Reply:
It is used in some type parameter things. I think it is a simpler vector version for use in the compiler before `Array` is compiled

Reply:
Maybe in this case it would be better to just duck type it and hope that it quacks like a vector.

Reply:
I just manually converted it to a Vector. (It does seem to quack exactly like a Vector, but it is not an AbstractVector and there is no `convert` to an AbstractVector)